RHIDP-6145: Added developer-focused content for software templates (#1098)
redhat-developer/red-hat-developers-documentation-rhdh: modules/customizing-templates/proc-searching-and-filtering-software-templates.adoc

// Module included in the following assemblies:
//
// * assemblies/assembly-configuring-templates.adoc

:_mod-docs-content-type: PROCEDURE
[id="proc-searching-and-filtering-software-templates_{context}"]
= Searching and filtering Software Templates in your {product} instance

You can search and filter for the Software Template that you want to use to create a new software component.

.Procedure

To search and filter for a Software Template, complete the following steps:

> . In your {product} instance, go to the *Home > Catalog* page, and click *Self-service*.

Reviewer: ```suggestion
. In the {product} navigation menu, click *Catalog* > *Self-service*.
```

> . Based on your requirement, select any option from any of the following dropdowns:

Reviewer: ```suggestion
. Based on your requirement, select any option from any of the following drop-down menus:
```

** *Categories*

> ** *Owner*

Reviewer: This renders fine, but I think these can be `*` rather than `**`
